prov/gni/src/gnix_av.c

```diff
@@ -856,6 +893,7 @@ DIRECT_FN int gnix_av_open(struct fid_domain *domain, struct fi_av_attr *attr,
 
 	av_priv->capacity = count;
 	if (type == FI_AV_TABLE) {
+		/* TODO: Use gnix_vector? */
```

Comment:
Don't leave TODOs in the code if you can help it. Open an issue and let's discuss there.